src/test/java/org/gridsuite/modification/server/modifications/VscCreationTest.java

```diff
@@ -0,0 +1,147 @@
+                .lossFactor(4F)
+                .minimumReactivePower(55.)
+                .voltageLevelId("v2")
+                .busOrBusbarSectionId("1.1")
+                .connectionName("top")
+                .connectionDirection(ConnectablePosition.Direction.TOP)
+                .reactiveCapabilityCurve(false)
+                .reactiveCapabilityCurvePoints(List.of())
+                .build();
+    }
+
+    private ConverterStationCreationInfos buildConverterStationWithReactiveCapabilityCurve() {
+        var point1 = ReactiveCapabilityCurveCreationInfos.builder()
+                .p(0.4)
+                .qmaxP(3.)
+                .qminP(0.)
+                .build();
+        var point2 = ReactiveCapabilityCurveCreationInfos.builder()
+                .p(0.6)
+                .qmaxP(2.)
+                .qminP(1.1)
+                .build();
+
+        return ConverterStationCreationInfos.builder()
+                .converterStationId("stationId1")
+                .converterStationName("station1")
+                .voltageRegulationOn(true)
+                .voltage(66.)
+                .lossFactor(40F)
+                .reactiveCapabilityCurve(true)
+                .reactiveCapabilityCurvePoints(List.of(point1, point2))
+                .voltageLevelId("v1")
+                .busOrBusbarSectionId("1.1")
+                .connectionName("top")
+                .connectionDirection(ConnectablePosition.Direction.TOP)
+                .build();
+    }
+
+    @Override
+    protected ModificationInfos buildModificationUpdate() {
+        return VscCreationInfos.builder()
+                .equipmentId("vsc1")
+                .equipmentName("vsc2Name")
+                .dcNominalVoltage(53.)
+                .dcResistance(2.)
+                .maximumActivePower(77.)
+                .p0(8.3F)
+                .operatorActivePowerLimitSide2(5.2F)
+                .convertersMode(HvdcLine.ConvertersMode.SIDE_1_RECTIFIER_SIDE_2_INVERTER)
+                .activePower(7.)
+                .operatorActivePowerLimitSide1(6.1F)
+                .operatorActivePowerLimitSide2(8.3F)
+                .angleDroopActivePowerControl(true)
+                .droop(2.1F)
+                .converterStation1(buildConverterStationWithMinMaxReactiveLimits())
+                .converterStation2(buildConverterStationWithReactiveCapabilityCurve())
+                .build();
+    }
+
+    @Override
+    protected void assertAfterNetworkModificationCreation() {
+        var list = getNetwork().getHvdcLineStream().toList();
+        assertNotNull(getNetwork().getHvdcLine("vsc1"));
+
+        assertEquals(1, getNetwork().getVoltageLevel("v1").getVscConverterStationStream()
+                .filter(transformer -> transformer.getId().equals("stationId1")).count());
+
+        assertEquals(1, getNetwork().getVoltageLevel("v2").getVscConverterStationStream()
+                .filter(transformer -> transformer.getId().equals("stationId2")).count());
+    }
```

Comment:
Add more checks on the the vsc characteristics, and on the vsc converter station 1 and station 2 characteristics ...